src/components/CodeBlock/index.tsx

```diff
@@ -7,6 +7,7 @@ import {
 import React from 'react';
 import copy from 'copy-to-clipboard';
 import { trackCopyClicks } from '../../utils/track';
+import { log } from 'console';
```

Comment:
I don't see this being used anywhere, if not can we remove the unused import.

Reply:
Good call, removed.